projects/1.19/assets/reliquary-v1-3/reliquary/lang/zh_cn.json

```
@@ -0,0 +1,395 @@
+
+  "item.reliquary.ice_magus_rod": "冰术士之杖",
+  "item.reliquary.ice_magus_rod.tooltip": "发射致伤的雪球。",
+  "item.reliquary.ice_magus_rod.tooltip2": "充能：%s",
+
+  "item.reliquary.infernal_chalice": "炼狱圣杯",
+  "item.reliquary.infernal_chalice.tooltip": "存储熔岩并保护你免受熔岩伤害",
+  "item.reliquary.infernal_chalice.tooltip2": "包含%smB熔岩",
+
+  "item.reliquary.infernal_claws": "炼狱之爪",
+  "item.reliquary.infernal_claws.tooltip": "以饥饿值为代价吸收火焰伤害。",
+  "item.reliquary.infernal_tear": "炼狱之泪",
+  "item.reliquary.infernal_tear.tooltip": "将物品烧炼成经验。",
+  "tooltip.reliquary.infernal_tear.absorb_unset": "右击移除物品关联",
+  "tooltip.reliquary.infernal_tear.tear_empty": "右击选取物品栏中最多的物品，使其可以被炼狱之泪烧炼。",
+  "item.reliquary.kraken_shell": "海妖壳甲",
+  "item.reliquary.kraken_shell.tooltip": "以饥饿度为代价吸收窒息伤害。",
+
+  "item.reliquary.lantern_of_paranoia": "诞妄提灯",
+  "item.reliquary.lantern_of_paranoia.tooltip": "自动在黑暗区域放置火把。Shift+右击激活。",
+  "item.reliquary.magicbane": "神魔克星",
+  "item.reliquary.magicbane.tooltip": "每条附魔都会增加其伤害。获得圣遗物中的掉落物的概率更高。",
+
+  "item.reliquary.magazines.empty_magazine": "空弹匣",
+  "item.reliquary.magazines.neutral_magazine": "中性弹匣",
+  "item.reliquary.magazines.exorcism_magazine": "驱魔弹匣",
+  "item.reliquary.magazines.blaze_magazine": "烈焰弹匣",
+  "item.reliquary.magazines.ender_magazine": "末影弹匣",
+  "item.reliquary.magazines.concussive_magazine": "震荡弹匣",
+  "item.reliquary.magazines.buster_magazine": "冲击弹匣",
+  "item.reliquary.magazines.seeker_magazine": "追踪弹匣",
+  "item.reliquary.magazines.sand_magazine": "扬沙弹匣",
+  "item.reliquary.magazines.storm_magazine": "风暴弹匣",
+
+  "item.reliquary.mercy_cross": "怜悯之十字架",
+  "item.reliquary.mercy_cross.tooltip": "对亡灵生物造成额外伤害。",
+
+  "item.reliquary.midas_touchstone": "迈达斯的点金石",
+  "item.reliquary.midas_touchstone.tooltip": "修复与黄金有关的物品",
+  "item.reliquary.midas_touchstone.tooltip2": "充能：%s",
+
+  "item.reliquary.phoenix_down": "凤凰之尾",
+  "item.reliquary.phoenix_down.tooltip": "原地满血复活一次，后会还原成天使之羽。",
+
+  "item.reliquary.empty_potion_vial": "浓缩药水瓶",
+  "item.reliquary.potion": "浓缩药水",
+  "item.reliquary.splash_potion": "喷溅型浓缩药水",
+  "item.reliquary.lingering_potion": "滞留型浓缩药水",
+  "item.reliquary.potion.tooltip": "效果：",
+
+  "item.reliquary.potion_essence": "药水精华",
+  "item.reliquary.potion_essence.tooltip": "效果：",
+
+  "item.reliquary.pyromancer_staff": "烈焰术士之杖",
+  "item.reliquary.pyromancer_staff.tooltip": "发射小火球（烈焰人）以及火球（恶魂），制造爆炸和火灾",
+  "item.reliquary.pyromancer_staff.tooltip.charges": "充能：%s",
+  "item.reliquary.pyromancer_staff.tooltip.blaze": "烈焰：%s",
+  "item.reliquary.pyromancer_staff.tooltip.controls": "Shift+滚轮改变模式",
+
+  "item.reliquary.rending_gale": "裂风者",
+  "item.reliquary.rending_gale.tooltip": "飞行，推拉生物，或者在雨天发射闪电",
+  "item.reliquary.rending_gale.tooltip2": "充能：%s",
+  "item.reliquary.rending_gale.tooltip.controls": "Shift+滚轮改变模式",
+
+  "item.reliquary.rod_of_lyssa": "犬牙钓竿",
+  "item.reliquary.rod_of_lyssa.tooltip": "能够钓取物品和偷窃的魔法钓竿。",
+
+  "item.reliquary.salamander_eye": "沙拉曼达之眼",
```

```suggestion
  "item.reliquary.salamander_eye": "沙罗曼达之眼",
```

```suggestion
  "item.reliquary.salamander_eye": "沙罗曼蛇之眼",
```
根据查到的情况比较常用的译名应该是这两者之一

---

倾向不改 https://zh.wikipedia.org/wiki/%E6%B2%99%E7%BE%85%E6%9B%BC%E9%81%94
官译蝾螈之眼就离谱了。